### DISCUSSION TITLE
How to implement a custom sub-form?

### DISCUSSION THREAD

**Discussion by Nilesh9768** [data-driven-forms/react-forms#1296] [Need help] [answered]
I'm trying to find a fully implemented example for custom sub-forms, But didn't get anything useful. Please provide a example for the same. Hoping for a quick response. Thank you!

**Comment by Hyperkid123** [comment, marked as answer]
@Nilesh9768 The SubForm component in mappers in our docs is just a simple layout component that is not connected to the form state.

You can take a look at this example of how to implement such a component: https://data-driven-forms.org/examples/custom-layout-component#heading-customlayoutcomponent

You can define and use any props you like.

**Reply by Nilesh9768** [reply]
@Hyperkid123 Thank you so much. This helps a lot.
